app/services/state_file/state_information_service.rb

```diff
@@ -42,6 +50,27 @@ def state_code_from_intake_class(klass)
         end
         state_code.to_s
       end
+
+      def intake_class_from_state_code(state_code)
+        STATES_INFO[state_code][:intake_class]
+      end
```

Comment:
this is also defined above as one of the simple getter methods. so accessing it would look like `StateInformationService.intake_class(state_code)`